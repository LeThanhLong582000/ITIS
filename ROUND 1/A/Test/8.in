1000
20400 1544 32165 28944 18811 9191 13961 29219 6463 14561 29452 12680 18054 4069 28082 28099 2968 22704 15251 14998 29689 23617 32516 31001 17458 21901 11635 5644 3796 17013 27795 619 6843 16629 19130 6723 32400 19614 22151 28101 5521 28742 22983 8344 12693 16382 15257 21364 11850 12100 19921 19437 18882 6010 2459 20108 26158 663 32609 24356 6377 3613 32238 23395 10936 31307 13523 9128 27984 14739 22025 17940 2584 5198 31449 24548 11825 13487 20473 10494 14260 12021 19704 24009 18490 23013 31451 31554 24738 2936 3593 333 27150 3537 22372 12104 20527 24689 20665 28390 4291 6574 18522 24891 25691 31841 28681 16915 25698 5600 21272 5512 21083 30764 30671 21460 25706 25707 12302 1891 15126 31128 23859 4397 25677 15225 11070 1858 4008 1489 1844 24998 22513 16344 21697 26976 17891 25576 10016 7630 28720 12153 320 1083 23753 3919 24863 5555 1881 7154 6778 16150 12562 14943 17577 10092 23981 13565 16325 12408 8620 24786 5387 26000 9699 8951 13957 19628 1078 27209 25063 19900 24996 29026 10935 9279 27013 23752 28592 3857 3599 2403 16233 12932 12831 7989 28722 24755 8247 15429 28478 15379 29378 19420 2824 26699 20896 24052 15599 15686 10462 4846 13859 12422 15557 28547 13253 26811 16151 11517 24138 12180 18837 22951 14369 13781 25028 23938 24469 23978 27121 14137 10228 32570 9075 1367 19953 23424 9776 12216 19271 12630 29027 17725 3487 10451 31132 25028 5300 2263 22536 12376 15285 27364 32498 23972 12779 24880 6026 3758 26267 17144 27877 12226 27694 4886 2118 12203 5775 13526 25244 31530 22741 25350 20299 19616 9762 7500 1071 9230 22986 32701 25310 31551 27320 14506 26414 2628 28174 4285 24312 12321 11532 14661 20251 21446 21043 21711 10691 20801 29098 28150 14827 18225 28896 20604 26656 22654 4467 30873 15496 13895 9041 3218 16932 1564 8733 5262 12114 30830 16852 25766 30614 9027 11952 27963 10905 29998 1100 25996 17522 21631 16394 1159 21285 16055 12978 15328 5481 31807 13659 10593 30383 15449 9615 12929 3167 11606 27818 32311 30821 25866 5763 16656 21740 11645 15182 3968 3433 2531 16577 10869 18652 1998 13942 31282 18771 12451 20881 11099 25775 12106 19362 8780 31402 32298 10445 14511 3097 12629 10536 31054 22383 12543 26980 20005 7811 30773 30384 21692 31123 11212 23002 19636 23432 7575 23991 21663 11131 4061 5318 26901 6009 19721 25104 23044 1155 26532 2386 10823 16345 13953 2654 3450 21881 24887 19391 30733 12258 4308 22684 30104 25885 6352 13791 14702 21758 11328 4757 7541 31912 29516 22424 821 9039 5347 23148 19246 17474 2222 14184 22127 23080 17730 5910 18020 28553 7833 25308 28400 6289 26819 11483 11963 14865 13017 20247 8654 16839 11827 1496 25181 13279 5594 2385 19427 12729 10816 27264 29813 24045 5423 14165 20559 16781 18944 10774 13236 5802 13837 26626 9973 7193 21865 4917 16591 22134 32718 13259 31481 25194 470 15432 13566 15402 22126 11576 22857 8256 5256 7186 22654 27022 25873 8546 28433 4240 5411 29223 26897 12257 6267 10362 22596 12801 4141 32625 28793 3157 10646 1603 32282 11119 27027 3757 11009 29840 14320 23188 1420 107 16164 19691 32244 29484 29755 8663 19930 5561 23315 29201 32291 5176 23243 1421 32518 1489 13945 14681 1635 8629 32283 31943 10327 17759 3025 10702 24077 12314 24905 15534 28375 6083 31098 11987 28541 31440 25237 30972 19413 21393 13368 156 3731 4278 13797 29824 23395 30482 18868 23731 20277 20638 28484 24991 15064 3265 29431 3700 27595 13774 7519 4814 30544 24527 30451 18322 15297 29825 29703 31512 3111 159 7419 18576 14857 26211 1714 32414 29058 6475 12101 22856 7062 18201 4857 4882 15046 4637 3439 4455 30178 16566 23323 12347 17695 1559 20215 28911 30058 18503 4217 4213 30203 1899 30146 18612 9260 19016 19992 16686 2444 13316 12438 29408 16431 10028 8994 873 29001 6652 11680 23407 8699 32730 11621 31807 17005 27683 31983 22707 13085 21034 17363 13657 13431 1662 758 21457 29861 25964 12544 28023 12200 17093 31502 16707 5544 1762 32168 14573 27528 7396 2546 26964 6590 5696 19238 29216 7522 10108 24641 166 3972 25139 15869 31245 14019 20575 27904 4271 22299 21720 30803 18626 13311 29975 8213 12344 30888 28736 9329 19939 31404 18272 15565 3037 2770 6898 3908 8487 18469 425 15447 16016 10567 15465 21215 729 12296 27542 16454 31254 31543 10642 4343 11351 10937 11905 5395 32648 18903 24912 15718 28822 13276 31221 17561 26580 9952 6392 30889 11420 30943 30770 14488 15417 8974 6357 31330 2052 5998 4780 27954 24270 5281 26788 3030 1406 27645 1676 13168 17147 28127 19481 4719 25421 28737 18733 958 30207 9103 3928 10811 19399 10679 15520 24683 30569 9088 9085 20588 21976 10268 13352 18475 24990 23296 2837 7639 21887 5107 18622 14485 25177 19813 5795 4469 31823 19715 32143 16548 806 14984 28956 27364 18927 6358 13433 9018 16871 21829 31851 5350 19611 24891 29330 2456 31056 2557 32238 8161 12705 708 29858 17896 6506 21780 6903 19550 9701 11081 5027 9652 12307 10611 9548 9611 1630 18531 31048 21612 16764 26374 14247 11193 24726 1522 9331 19939 28842 24895 8758 3806 17198 23704 15805 24786 30710 30097 10012 30996 24151 12347 15346 16724 29069 12791 1774 26904 29786 32391 22674 11113 28656 2923 23863 7371 6673 7530 21586 5491 26229 9327 19701 2997 18284 2454 2969 29372 25028 9095 32026 29053 14317 17667 23299 1517 4187 6588 25868 26738 6956 17770 21473 17304 6923 31003 26917 5126 5460 29386 16567 17584 19832 6964 18228 23374 4004 13799 31165 18700 3741 22428 21242 13187 22767 5830 2398 18920 10566 10859 685 31363 5660 27443 8380 7267 25748 27327 19321 24913 678 5435 23594 9218 27773 4407 19962 26257 285 20914 22449 5420 2633 3704 21187 24845 31652 5807 5004 18347 11542 12716 5850 6706 5233 523 10412 6449 16981 7934 26681 30867 9809 22501 19764 24529 24138 22047 4687 14950 5409 4384 10494 27778 13456 18628 6781 16386 28245 31203 31252 7584 24723 294 19610 26215 22901 24601 