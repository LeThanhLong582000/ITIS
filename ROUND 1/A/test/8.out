41169202007413500